misk-zookeeper-testing/src/main/kotlin/misk/zookeeper/testing/ZkTestModule.kt

```diff
@@ -57,7 +59,8 @@ class ZkTestModule(
     }
 
     companion object {
-      private val sharedZookeeper = EmbeddedZookeeper(zkPortKey)
+      private val sharedZookeeper = EmbeddedZookeeper(
+          zkPortKey)
```

Comment:
/nit one line

Reply:
weird. intellij did this during the refactor. bad intellij.